### PR TITLE
Update test to accept channel from test args

### PR DESCRIPTION
**Pull request by mdrahman-suse**
#### Testing ####

Checklist:
1. If your PR changes anything on or related to Jenkins, run it pointing to your branch to make sure it's okay.

2. Verify code lint; we should not have errors.

3. Update the documentation if needed.

4. Update makefile and docker run if adding new tests.

5. Run your tests at least 4 times with all configurations needed and possible.

6. If needed test with different os types.

### CODE REVIEW

**Comment by fmoral2** [comment]
nice catch!

you will probably need to update `/scripts/test_runner.sh` as well

File: pkg/testcase/upgradenodereplacement.go [hunk 1]
```diff
@@ -74,7 +78,8 @@ func TestUpgradeReplaceNode(cluster *shared.Cluster, version string) {
 
 func nodeReplaceAgents(
 	cluster *shared.Cluster,
-	version string,
+	version,
```
Comment (ShylajaDevadiga) [inline pkg/testcase/upgradenodereplacement.go:81]: have channel in same line?

File: pkg/testcase/upgradenodereplacement.go [hunk 2]
```diff
@@ -287,7 +292,8 @@ func joinRemainServers(
 	oldServerIPs []string,
 	serverLeaderIp,
 	token,
-	version string,
+	version,
+	channel string,
```
Comment (ShylajaDevadiga) [inline pkg/testcase/upgradenodereplacement.go:296]: same. add to line above with version

File: pkg/testcase/upgradenodereplacement.go (hunk 1 above)
Comment (mdrahman-suse) [reply pkg/testcase/upgradenodereplacement.go:81]: I think its better keep the vars in separate line

File: pkg/testcase/upgradenodereplacement.go (hunk 2 above)
Comment (mdrahman-suse) [reply pkg/testcase/upgradenodereplacement.go:296]: I think its better keep the vars in separate line

**Review by fmoral2 — APPROVED** [review]

**Pull request merged**